packages/models/src/token-generation-readmodel/platform-states-entry.ts

```diff
@@ -64,6 +65,7 @@ export const PlatformStatesClientEntry = PlatformStatesBaseEntry.extend({
   clientKind: ClientKindTokenGenStates,
   clientConsumerId: TenantId,
   clientPurposesIds: z.array(PurposeId),
+  adminId: UserId.optional(),
```

Comment:
Each field in entry using client* prefix, what do you think if we follow the same patter for the new one?
``` suggestion
  clientAdminId: UserId.optional(),
```